src/Common/src/System/ThrowHelper.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    internal static class ThrowHelper
```

Comment:
I do not think that the ThrowHelper should be common. It should be always local to the project to avoid bloating every assembly with every other assembly exception messages.

Reply:
Ok